collections/_posts/2023-11-09-Elongated-structures.md

```diff
@@ -0,0 +1,355 @@
+- Minimum distance : only the estimator with the smallest point-to-line distance is used for evaluating the distance-dependent function. Easily parallelized, it allows the heatmap generation algorithm to be executed at linear time complexity
+- Inverse distance weighting (IDW) : allows the contribution of all distance estimates.
+
+**Selection of the distance-dependant function**
+- frequently observed signal/noise shapes (Gaussian, Laplace, cosine)
+- versatile in their encoding characteristics (triangular, rectangular)
+- custom signal shape in the form of a simulated hollow-core catheter profile
+
+**Extension to 3D**
+The local hull is approximated by extending the two-dimensional bounding boxes to right rectangular prisms.
+
+## Signal extraction and evaluation metrics
+
+Two post-processing protocols for signal recovery and evaluation, with specific curve type and dataset : 
+
+### Center-line extraction and ASSD comparison
+
+Signal estimation as a set of point + post-processing the estimation : 
+- turn it to a binary image (using Otsu’s method, automatic image threshpolding)
+- transform into a curve with uniform width using medial axis transformation
+
+ASSD between two curves $$C$$ and $$C'$$, 
+- minimum euclidiean distance of point $$x$$ in curve $$C$$ to point $$x'$$ on curve $$C'$$ :
+$$d(x, C') = \underset{x′ \in C′}{min} \vert \vert x − x' \vert \vert \underset{2}{}$$ 
+- evaluating this distance for every point $$x \in C$$ and $$x' \in C'$$ and averaging the resulting distance values :  
+
+$$ASSD = \frac{1}{|C| + |C'|}*(\underset{x \in C}{\Sigma}d(x, C') +\underset{x' \in C'}{\Sigma}d(x, C')d(x', C) )$$ 
+
+Advantage : invariant to the width and distribution of the simulated signal and estimated heatmap
+
+Limitations :
+- noisy and self-intersecting heatmap estimates, the thinning can cause branching artifacts 
+- larger signal or heatmap width, longer subsidiary branch
+- effect is negligible as most errors are caused by signal fragmentation in ambiguous regions
+
+### Non-maximum suppression of the heatmap, morphological thinning and F-measure
+
+Interpret the heatmap as a noisy curve probability map :
+- refine the curve estimation using non-maximum suppression (NMS), keep the stronger indicator of a curve’s presence within a certain radius $$r_{spr}$$
+- convert probability in heatmap to binary curve label with a probability threshold
+- iterative thinning to get a curve with uniform width
+
+Calculate the minimum-cost correspondence (Euclidiean distance) between point on the two curve. Then calculate Precision and Recall to compute F-measure $$F=\frac{2PR}{P+R}$$. Assess how well the estimated curve matches the real one.Select the best curve threshold across all images (either at optimal dataset scale ODS or optimal image scale OIS &rarr; ensuring different scale consideration)
+
+## Neural network architecture 
+network's parameters are initialized using the He initialization strategy
```

Comment:
I think this kind of things is implementation details and could be removed for a post (c.f. our discussion on the "synthetic" aspect of biblio review)